### PR TITLE
Feat redux ex1

### PR DESCRIPTION
**Pull request by e-schultz**
@winkerVSbecks @bennett000 

still very rough, but wondering if I'm making this more complex than it needs to be. 

What I'm thinking as the overall exercise:

* implement the `ngRedux` on the app container
* implement the buildRating on rating-component 
* probably leave all of the templates as-is, don't gut those out
* implement the action creator for updating a skill level
* implement the reducer to handle the skill update
* stretch: hook up adding a new person?

On the other hand, tempted to restart with a basic todo app with just add/remove  items. 

Sort of went on a tangent playing around with passing down events / emitting them back up. ie: how can I have the rating simply emit the index of what was clicked on - and passing it back up the component tree.

* rating just emits the index/rating that was clicked - doesn't care about the skill/person
* skill-rating gets that, and emits the skill + rating, and not needing to care about the person
* the app-container then gets the event from the skill-rating, then knows person + skill + new rating for the final dispatch to redux (not hooked up yet).

Some of the naming conventions could be cleaned up. Sort of turned into my own experiment of "whats the best way of trying to pass things down without needing to make the inbetween components know about more than they should".

### CODE REVIEW

**Comment by winkerVSbecks** [comment]
@e-schultz I like this example. The ToDo stuff is already there in the sample app. It demonstrates the importance of immutability and working with immutable lists. This should allow you to limit the number of exercises needed in the Immutable.js section.

Like you said, add a new person can be an optional stretch goal for when we have time/for people who are more advanced.

**Comment by e-schultz** [comment]
@winkerVSbecks updated a few things and have the reducer handling the action now

**Comment by winkerVSbecks** [comment]
:+1:

**Comment by e-schultz** [comment]
@bennett000 @winkerVSbecks 

Decided to create a top / bottom skill container and added the skeleton for them to complete.

Do the instructions/etc make sense?

File: exercises/redux/1.0-Redux/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+* Using mapping application state to target, and doing data transformation
+* Hooking up action creators
+* Illustrating the distinction between containers and dumb components
+
+
+# To be completed:
+
+* Complete _*app/compoentns/skill-label.ts*_ 
+* Implement the updateSkill action oin _*app/actions/people-actions.ts*_
```
Comment (winkerVSbecks) [inline exercises/redux/1.0-Redux/README.md]: typo _oin_?

**Comment by winkerVSbecks** [comment]
LGTM :+1:

**Comment by e-schultz** [comment]
fixed typo

**Pull request merged**